diff --git a/config.def.h b/config.def.h
index 6f05dce..d19e599 100644
--- a/config.def.h
+++ b/config.def.h
@@ -93,33 +93,39 @@ char *termname = "st-256color";
  */
 unsigned int tabspaces = 8;
 
-/* Terminal colors (16 first used in escape sequence) */
+/* Terminal colors (16 first used in escape sequence)
+ * Base16 PaperColor Light scheme by Jon Leopard (http://github.com/jonleopard) based on PaperColor Theme (https://github.com/NLKNguyen/papercolor-theme)
+ */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	[0] = "#eeeeee", /* base00 */
+	[1] = "#bcbcbc", /* base08 */
+	[2] = "#8700af", /* base0B */
+	[3] = "#d70087", /* base0A */
+	[4] = "#d75f00", /* base0D */
+	[5] = "#005faf", /* base0E */
+	[6] = "#d75f00", /* base0C */
+	[7] = "#444444", /* base05 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	[8] = "#5f8700", /* base03 */
+	[9] = "#bcbcbc", /* base08 */
+	[10] = "#8700af", /* base0B */
+	[11] = "#d70087", /* base0A */
+	[12] = "#d75f00", /* base0D */
+	[13] = "#005faf", /* base0E */
+	[14] = "#d75f00", /* base0C */
+	[15] = "#878787", /* base0F */
+
+	/* extra base16 colors */
+	[16] = "#d70000", /* base09 */
+	[17] = "#005f87", /* base0F */
+	[18] = "#af0000", /* base01 */
+	[19] = "#008700", /* base02 */
+	[20] = "#0087af", /* base04 */
+	[21] = "#005f87", /* base06 */
 
 	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
 };
 
 
diff --git a/x.c b/x.c
index 120e495..da3ec4b 100644
--- a/x.c
+++ b/x.c
@@ -754,7 +754,7 @@ xloadcolor(int i, const char *name, Color *ncolor)
 	XRenderColor color = { .alpha = 0xffff };
 
 	if (!name) {
-		if (BETWEEN(i, 16, 255)) { /* 256 color */
+		if (BETWEEN(i, 22, 255)) { /* 256 color */
 			if (i < 6*6*6+16) { /* same colors as xterm */
 				color.red   = sixd_to_16bit( ((i-16)/36)%6 );
 				color.green = sixd_to_16bit( ((i-16)/6) %6 );
