diff --git a/config.def.h b/config.def.h
index 6f05dce..d19e599 100644
--- a/config.def.h
+++ b/config.def.h
@@ -93,33 +93,39 @@ char *termname = "st-256color";
  */
 unsigned int tabspaces = 8;
 
-/* Terminal colors (16 first used in escape sequence) */
+/* Terminal colors (16 first used in escape sequence)
+ * Base16 darkmoss scheme by Gabriel Avanzi (https://github.com/avanzzzi)
+ */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	[0] = "#171e1f", /* base00 */
+	[1] = "#ff4658", /* base08 */
+	[2] = "#499180", /* base0B */
+	[3] = "#fdb11f", /* base0A */
+	[4] = "#498091", /* base0D */
+	[5] = "#9bc0c8", /* base0E */
+	[6] = "#66d9ef", /* base0C */
+	[7] = "#c7c7a5", /* base05 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	[8] = "#555e5f", /* base03 */
+	[9] = "#ff4658", /* base08 */
+	[10] = "#499180", /* base0B */
+	[11] = "#fdb11f", /* base0A */
+	[12] = "#498091", /* base0D */
+	[13] = "#9bc0c8", /* base0E */
+	[14] = "#66d9ef", /* base0C */
+	[15] = "#e1eaef", /* base0F */
+
+	/* extra base16 colors */
+	[16] = "#e6db74", /* base09 */
+	[17] = "#d27b53", /* base0F */
+	[18] = "#252c2d", /* base01 */
+	[19] = "#373c3d", /* base02 */
+	[20] = "#818f80", /* base04 */
+	[21] = "#e3e3c8", /* base06 */
 
 	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
 };
 
 
diff --git a/x.c b/x.c
index 120e495..da3ec4b 100644
--- a/x.c
+++ b/x.c
@@ -754,7 +754,7 @@ xloadcolor(int i, const char *name, Color *ncolor)
 	XRenderColor color = { .alpha = 0xffff };
 
 	if (!name) {
-		if (BETWEEN(i, 16, 255)) { /* 256 color */
+		if (BETWEEN(i, 22, 255)) { /* 256 color */
 			if (i < 6*6*6+16) { /* same colors as xterm */
 				color.red   = sixd_to_16bit( ((i-16)/36)%6 );
 				color.green = sixd_to_16bit( ((i-16)/6) %6 );
