diff --git a/config.def.h b/config.def.h
index 6f05dce..d19e599 100644
--- a/config.def.h
+++ b/config.def.h
@@ -93,33 +93,39 @@ char *termname = "st-256color";
  */
 unsigned int tabspaces = 8;
 
-/* Terminal colors (16 first used in escape sequence) */
+/* Terminal colors (16 first used in escape sequence)
+ * Base16 Google Dark scheme by Seth Wright (http://sethawright.com)
+ */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	[0] = "#1d1f21", /* base00 */
+	[1] = "#cc342b", /* base08 */
+	[2] = "#198844", /* base0B */
+	[3] = "#fba922", /* base0A */
+	[4] = "#3971ed", /* base0D */
+	[5] = "#a36ac7", /* base0E */
+	[6] = "#3971ed", /* base0C */
+	[7] = "#c5c8c6", /* base05 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	[8] = "#969896", /* base03 */
+	[9] = "#cc342b", /* base08 */
+	[10] = "#198844", /* base0B */
+	[11] = "#fba922", /* base0A */
+	[12] = "#3971ed", /* base0D */
+	[13] = "#a36ac7", /* base0E */
+	[14] = "#3971ed", /* base0C */
+	[15] = "#ffffff", /* base0F */
+
+	/* extra base16 colors */
+	[16] = "#f96a38", /* base09 */
+	[17] = "#3971ed", /* base0F */
+	[18] = "#282a2e", /* base01 */
+	[19] = "#373b41", /* base02 */
+	[20] = "#b4b7b4", /* base04 */
+	[21] = "#e0e0e0", /* base06 */
 
 	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
 };
 
 
diff --git a/x.c b/x.c
index 120e495..da3ec4b 100644
--- a/x.c
+++ b/x.c
@@ -754,7 +754,7 @@ xloadcolor(int i, const char *name, Color *ncolor)
 	XRenderColor color = { .alpha = 0xffff };
 
 	if (!name) {
-		if (BETWEEN(i, 16, 255)) { /* 256 color */
+		if (BETWEEN(i, 22, 255)) { /* 256 color */
 			if (i < 6*6*6+16) { /* same colors as xterm */
 				color.red   = sixd_to_16bit( ((i-16)/36)%6 );
 				color.green = sixd_to_16bit( ((i-16)/6) %6 );
